dashboard/app/models/levels/blockly.rb

```diff
@@ -617,6 +620,32 @@ def shared_functions
     end.join
   end
 
+  def localize_behaviors(block_xml)
+    block_xml.xpath("//block[@type=\"gamelab_behavior_get\"]").each do |behavior|
+      behavior.xpath(".//title[@name=\"VAR\"]").each do |parameter|
+        next unless parameter.content == 'this sprite'
```

Comment:
nit: use `I18n.t('behaviors.this_sprite', locale: :en)` (i think that should work directly, but you should double-check me) just in case the string changes

same on lines 633 and 637 below